snorkel/udf.py

```diff
@@ -1,5 +1,5 @@
 from multiprocessing import Process, JoinableQueue
-from Queue import Empty
+from queue import Empty
```

Comment:
I added `future` in the package requirements. With that installed, this code works under Python 2.